test/active_support_test_case_helper.rb

```diff
@@ -39,6 +40,10 @@ def skip_if_plugin_asked_to
     end
   end
 
+  def fetch_proxy_features
+    ProxyAPI::Features.any_instance.stubs(:features => Feature.name_map.keys)
```

Comment:
Yeah, it was a precaution for Katello tests. I will open a PR on the failing tests there (to use factories)